quinn-proto/src/connection/mod.rs

```diff
@@ -1371,13 +1393,21 @@ impl Connection {
             }
         }
 
+        let mut timestamp_iter = self.config.ack_timestamp_config.as_ref().map(|cfg| {
+            let decoder = ack.timestamp_iter(cfg.basis, cfg.exponent.0).unwrap();
```

Comment:
Should at least have a comment about why this `unwrap()` is safe.

Reply:
I fixed it so it does a `if let Some` check